_gsocproposals/proposal_ATLASEigen.md

```diff
@@ -0,0 +1,39 @@
+---
+title:  Faster matrix algebra for ATLAS
+layout: gsoc_proposal
+project: ATLAS
+organization:
+  - RAL
```

Comment:
RAL will need to be added as a new organisation under the CERN-HSF umbrella, could you please follow the instructions [here](http://hepsoftwarefoundation.org/gsoc/guideline.html) on that regard?

Reply:
Nevermind, just saw you already did that :)